src/pip/_internal/commands/install.py

```diff
@@ -459,11 +459,13 @@ def run(self, options, args):
 
         if options.target_dir:
             self._handle_target_dir(
-                options.target_dir, target_temp_dir, options.upgrade
+                options.target_dir, target_temp_dir, options.upgrade,
+                options.prefix_path
```

Comment:
Doesn't this *only* handle `--prefix=""` in the case where `--target` is specified? What about other cases where the user might want to reset a globally-set prefix?